src/st/st0/3D8F0.c

```diff
@@ -131,13 +138,23 @@ void func_801BD8F0(Entity* self) {
             prim = prim->next;
         }
         prim = self->ext.prim;
+#endif
         for (i = 0; i < 15; i++) {
             for (j = 0; j < 15; j++) {
                 xVar = j * 17;
                 yVar = i * 16;
+#ifdef VERSION_PSP
+                prim->tpage = ((yVar & 0x100) >> 4 | 0x100 |
+                               ((xVar + 0x300) & 0x3FF) >> 6) |
+                              ((yVar & 0x200) << 2);
```

Comment:
I have no idea what this mess is doing, if anyone has a way to simplify this please let me know